xrpl/utils/tx_parser.py

```diff
@@ -0,0 +1,277 @@
+import pydash
```

Comment:
This new package needs to be added to the poetry dependencies. Instructions for how to use that are in `CONTRIBUTING.md`.